lib/services/elasticsearch.js

```diff
@@ -542,21 +542,28 @@ module.exports = function (kuzzle, options) {
   };
 
   /**
-   * Reset all indexes
+   * Reset all indexes that the users is allowed to
    *
    * @return {Promise}
    */
   this.deleteIndexes = function (requestObject) {
+    var
+      allowedIndexes = requestObject.data.body.indexes,
+      deletedIndexes = [];
+
     delete requestObject.data.body;
 
     return this.listIndexes(requestObject)
```

Comment:
yes, because it's not exactly the same: the adminController list all indexes that the user is allowed to READ.
Here, we need the indexes that the user is allowed to delete.